src/text/0-shortcuts.tex

```diff
@@ -24,3 +24,4 @@
 \newcommand{\gammaray}{$\gamma$-ray~}
 \newcommand{\gammarays}{$\gamma$-rays~}
 \newcommand{\gadf}{GADF~}
+\newcommand{\milagro}{MILAGRO~}
```

Comment:
These commands wl result in an unwanted space if these happen to come at the end of a sentence. 

You can use https://ctan.org/pkg/xspace?lang=de or just make sure to use `\milagro{}` instead of `\milagro`